octygon-button-styler.php

```diff
@@ -1,146 +1,171 @@
 <?php
 /**
-* Plugin Name: Octygon Button Styler
-* Plugin URI: http://octygon.com/
-* Description: Takes the already amazing Divi buttons to the next level.
-* Version: 1.0
-* Author: Rob Stinson
-* Author URI: http://octygon.com
+ * Plugin Name: Octygon Button Styler
+ * Plugin URI: http://octygon.com/
+ * Description: Takes the already amazing Divi buttons to the next level.
+ * Version: 1.0
+ * Author: Rob Stinson
+ * Author URI: http://octygon.com
  * License: GPLv2 or later
-*/
+ */
 
+/**
+ * Add css button styles.
+ *
+ * @action wp_enqueue_scripts
+ */
 function octygon_button_styler_assets() {
 	wp_enqueue_style( 'octygon-button-styles', plugins_url( '/style.css', __FILE__ ) );
-	// wp_enqueue_script( 'octygon-button-scripts', plugins_url( '/scripts.js', __FILE__ ) );
 }
 add_action( 'wp_enqueue_scripts', 'octygon_button_styler_assets' );
 
+/**
+ * Register customizer controls.
+ *
+ * @action customize_register
+ *
+ * @param WP_Customize_Manager $wp_customize
+ */
 function oct_customize_register( $wp_customize ) {
-  	$wp_customize->add_setting( 'oct_btn_grow_styles',
+	$wp_customize->add_setting( 'oct_btn_grow_styles',
```

Comment:
There were some consistent spacing problems with these settings. Key takeaways are:

- Always indent with tabs
- Align array values with spaces (not tabs), and only to the longest key
- The last item in an array should always include a comma afterwards (although this isn't required by PHP, it's a WPCS)
- Don't let there be an extra spaces at the end of a line.